rdtools/normalization.py

```diff
@@ -381,3 +381,37 @@ def check_series_frequency(series, series_description):
     else:
         freq = series.index.freq
     return freq
+
+
+def energy_from_power(power_series, max_interval_hours):
```

Comment:
```suggestion
def energy_from_power(power_series, max_interval_hours = None):
```

Reply:
[pep8 - no spaces around equals for default keyword args](https://pep8.org/#other-recommendations)
>Don’t use spaces around the = sign when used to indicate a keyword argument or a default parameter value.
>
>Yes:

```python
def complex(real, imag=0.0):
    return magic(r=real, i=imag)
```
>No:

```python
def complex(real, imag = 0.0):
    return magic(r = real, i = imag)
```

Reply:
Fixed, thanks!